apis/servicenetworking/v1beta1/referencers.go

```diff
@@ -1,62 +0,0 @@
-/*
```

Comment:
I think this file should stay. Made the same mistake while trying to make angryjet work since it requires code to be compilable.